src/mouse.rs

```diff
@@ -57,8 +58,12 @@ fn get_inputs<'a>(
     let cursor_latest = match cursor.iter().last() {
         Some(cursor_moved) => {
             if let RenderTarget::Window(window) = camera.target {
-                if cursor_moved.id == window {
-                    Some(cursor_moved.position)
+                if let WindowRef::Entity(camera_entity) = window {
+                    if cursor_moved.window == camera_entity {
+                        Some(cursor_moved.position)

```

Comment:
This is the main section causing issues due to the changes bevy 0.10 made to how windows are handled. The `cursor_moved.window` handle points to the window entity, but extracting the `WindowRef` from the `camera_target` returns `WindowRef::Primary` instead of `WindowRef::Entity`.

Reply:
I've tried removing the nested checks entirely, and just returning `Some(cursor_moved.position`. This actually worked, but the rays seemed to be casting _towards_ the camera from the world, and not from the camera like it's supposed to.

Reply:
You could query for the `PrimaryWindow` and then if `WindowRef::Primary` use that entity.
The problem about the rays being reversed, is caused by the default implementation of FocusPolicy which got changed in https://github.com/bevyengine/bevy/pull/7161.

Both are fixes in this commit: https://github.com/aevyrie/bevy_mod_picking/commit/aaf439f5ef645fae36e98969b898e52aa4b44812